src/strands/tools/mcp/mcp_client.py

```diff
@@ -41,6 +41,11 @@
     "image/webp": "webp",
 }
 
+CLIENT_SESSION_NOT_RUNNING_ERROR_MESSAGE = (
+    "the client session is not running. Please ensure the agent is used within "
```

Comment:
Nit - remove the please

```suggestion
    "the client session is not running. Ensure the agent is used within "
```

We could also link to the docs for extra clarity (I always like it when I get links explaining proper usage) but that might be overkill here

Reply:
committed suggestion, I'm not sure the docs will help this problem the users explained that they were reading the docs directly. I do think there is another improvement that can be made though looking at this again 

https://github.com/strands-agents/docs/blob/main/docs/user-guide/concepts/tools/mcp-tools.md?plain=1#L44-L49

The lines here show the agent creation but I think the docs should be updated to have an example not just creating the agent, but calling it. Will follow up with this on the other package.